airflow/ui/src/pages/Dashboard/HealthSection.tsx

```diff
@@ -16,7 +16,10 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { Skeleton, Tag, TagLabel, Text, Tooltip } from "@chakra-ui/react";
+import { Skeleton, TagLabel, Text } from "@chakra-ui/react";
+
+import { Tag } from "src/components/ui/Tag";
+import { Tooltip } from "src/components/ui/Tooltip";
```

Comment:
WDYT about adding a `src/components/ui/index.ts` export file to make imports from the snippets cleaner?  It does come with a little extra overhead, but I imagine these won't be constantly changing.
```suggestion
import { Tag, Tooltip } from "src/components/ui/";
```